src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return Color.values()[(new Random()).nextInt(Color.values().length)].name();
```

Comment:
Don't create new Random every time this method is invoked. Move Random to a class field